Makefile

```diff
@@ -0,0 +1,154 @@
+                          btf.h \
+                          bpf_helpers.h \
+                          bpf_helper_defs.h \
+                          bpf_tracing.h \
+                          bpf_endian.h \
+                          bpf_core_read.h \
+                          libbpf_common.h \
+                          usdt.bpf.h
+
+    LIBBPF_SYS_LIBBPF_BASE_PATH := $(dir $(shell cargo metadata --format-version=1 | jq -r '.packages | .[] | select(.name == "libbpf-sys") | .manifest_path'))
+    LIBBPF_SYS_LIBBPF_INCLUDES :=  $(wildcard $(addprefix $(LIBBPF_SYS_LIBBPF_BASE_PATH)/libbpf/src/, $(LIBBPF_API_HEADERS)))
+    LIBBPF_INCLUDES := $(ROOT_DIR)/src/.out
+    LIBBPF_INCLUDE_FILES := $(addprefix $(LIBBPF_INCLUDES)/bpf/,$(LIBBPF_API_HEADERS))
+endif
+
+FILTER_INCLUDES := src/core/filters/packets/bpf/include \
+                   src/core/filters/meta/bpf/include
+INCLUDES_ALL := $(abspath $(wildcard src/core/probe/bpf/include \
+                                     src/core/probe/kernel/bpf/include \
+                                     src/core/probe/user/bpf/include \
+                                     src/core/events/bpf/include \
+                                     src/core/tracking/bpf/include \
+                                     /usr/include/x86_64-linux-gnu \
+                                     vendor/linux/include \
+                                     vendor/linux/asm/x86/include \
+                                     $(FILTER_INCLUDES)))
+INCLUDES_ALL += $(LIBBPF_INCLUDES)
+
+OVS_INCLUDES := $(abspath src/module/ovs/bpf/include)
+INCLUDES := $(addprefix -I, $(INCLUDES_ALL))
+
+EBPF_PROBES := $(abspath src/core/probe/kernel/bpf \
+                        src/core/probe/user/bpf)
+
+GENERIC_HOOKS := $(abspath src/module/skb/bpf \
+                           src/module/skb_drop/bpf \
+                           src/module/skb_tracking/bpf \
+                           src/module/nft/bpf \
+                           src/module/ct/bpf)
+
+OVS_HOOKS := $(abspath src/module/ovs/bpf)
+OUT_NAME := HOOK
+
+JOBS := $(patsubst -j%,%,$(filter -j%,$(MAKEFLAGS)))
+
+ifneq ($(JOBS),)
+    CARGO_JOBS := $(JOBS)
+else
+    CARGO_JOBS := 1
+endif
+
+all: build
+
+install: build
+	$(CARGO) $(CARGO_COMMON_OPTS) install $(CARGO_CMD_OPTS)
+
+release: CARGO_CMD_OPTS += --release
+release: build
+
+build: ebpf
+	$(call out_console,CARGO,building retis ($(if $(findstring --release,$(CARGO_CMD_OPTS)),release,debug))...)
+	$(Q)CARGO_BUILD_JOBS=$(CARGO_JOBS) \
+	RETIS_PKG_VERSION=$(RELEASE_VERSION) \
+	RETIS_RELEASE_NAME=$(RELEASE_NAME) \
+	$(CARGO) $(CARGO_COMMON_OPTS) build $(CARGO_CMD_OPTS)
+
+test: ebpf
+	$(call out_console,CARGO,running tests ...)
+	$(Q)CARGO_BUILD_JOBS=$(CARGO_JOBS) \
+	$(CARGO) $(CARGO_COMMON_OPTS) test $(CARGO_CMD_OPTS)
+
+bench: ebpf
+	$(call out_console,CARGO,building benchmarks ...)
+	$(Q)CARGO_BUILD_JOBS=$(CARGO_JOBS) \
+	$(CARGO) $(CARGO_COMMON_OPTS) build -F benchmark $(CARGO_CMD_OPTS)
```

Comment:
\+ `--release`

Reply:
looking at our current CI, we don't add `--release`.
Maybe for this target, if we want a `benchmark` compiled as with the release flag we could use `$(CARGO_CMD_OPTS)`?

Reply:
> looking at our current CI, we don't add `--release`.

Well, this is a bug. The benchmark command really must be compiled as a release binary.